src/app/core/api/portal-api.service.ts

```diff
@@ -243,6 +244,31 @@ export class PortalApi extends BaseApi {
     getAssetProfiles(): Observable<IAssetProfile[]> {
         return this.get<IAssetProfile[]>(`${this._apiUrl}/assetprofile/get`);
     }
+
+    getPlantInformations(): Observable<IPlantInformation[]> {
+        return this.get<IPlantInformation>(`${this._apiUrl}/plantinformation`);
+    }
+
+    getPlantInformation(id: number): Observable<IPlantInformation> {
+        return this.get<IPlantInformation>(`${this._apiUrl}/plantinformation/${id}`);
+    }
+
+    updatePlantInformation(data: any): Observable<any> {
+        return this.put<any>(`${this._apiUrl}/plantinformation`, data);
+    }
+
+    createPlantInformation(data: any): Observable<any> {
+        return this.post<any>(`${this._apiUrl}/plantinformation`, data);
+    }
```

Comment:
Have you checked the branch I gave? this should be plant-information

Reply:
Kindly see update